tests/phpunit/tests/formatting/sanitizePost.php

```diff
@@ -26,6 +26,9 @@ public function test_int_fields() {
 				case 'string':
 					$this->assertIsString( $post->$field, "field $field" );
 					break;
+				default:
+					$this->fail( "Type $type is not handled by this test." );
+					break;
```

Comment:
This seems redundant as the `$int_fields` array is defined only just above and only contains `integer` and `string` types.

Reply:
Correct but this protects against an unexpected value being added to that array, either intentionally or unintentionally, which would otherwise not cause the test to fail.